#167749946 Setup Swagger

davidteejay opened the pull request:
### What does this PR do?
Sets up Swagger Documentation for the project

### Description of Task to be completed?
Created the `swagger.json` file and set up the `/docs` route for viewing the documentation

### How should this be manually tested?
To test:
- Visit the /docs routes in a browser

### What are the relevant pivotal tracker stories?
[#167749946](https://www.pivotaltracker.com/story/show/167749946)

hustlaviola commented on package.json:
```diff
@@ -25,6 +25,8 @@
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
+    "swagger-jsdoc": "^3.4.0",
```
This package seems redundant, you might want to remove it

hustlaviola reviewed (changes requested):
Awesome. This is ready to go except for the es5 syntax. Please fix that. Good job

davidteejay commented:
The develop branch is in es5 syntax

alatos2 committed:
chore(setup): Setup Swagger

- Installed swagger-ui-express
- Created swagger.json file
- Created the /docs routes for the documentation
- Modified build command to copy swagger.json file to build folder

[Finishes #167749946]

hustlaviola reviewed (approved):
Good job

ascii-dev reviewed (approved):
LGTM
